stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -45,6 +45,9 @@ spec:
         - name: config
           mountPath: /fluent-bit/etc/fluent-bit.conf
           subPath: fluent-bit.conf
+        - name: config
```

Comment:
The Chart.yaml version will need to be bumped as well

Reply:
Sorry, pushed too quickly. This has been fixed.